tensorflow/python/eager/def_function_test.py

```diff
@@ -938,6 +938,17 @@ def f(x):
 
     self.assertLen(logs.output, 1)
     self.assertIn('Tracing is expensive', logs.output[0])
+  
+  def test_experimental_get_tracing_count(self):
```

Comment:
Since the tracing counter is global, it would be useful to test it on a method as well. Specifically:

```
class TestClass:
  @tf.function
  def method(...):
    ...

obj1 = TestClass()
# cause some retraces on obj1.method...
# verify counters for obj1.method

obj2 = TestClass()
# cause some retraces on obj2.method...
# verify counters for obj2.method
# then verify counters for obj1.method again
```